reference/docs-conceptual/learn/deep-dives/everything-about-arrays.md

```diff
@@ -839,7 +839,11 @@ At this point, you're starting to wonder how to add items to an array. The quick
 can't. An array is a fixed size in memory. If you need to grow it or add a single item to it, then
 you need to create a new array and copy all the values over from the old array. This sounds
 expensive and like a lot of work, however, PowerShell hides the complexity of creating the new
```

Comment:
What about something alonge the lines of:
```
This sounds expensive and like a lot of work. However, PowerShell hides the complexity of creating the new	array by implementing the addition operator (`+`). PowerShell does not implement a
subtraction operation.
```

Just seems to seem simpler but I don't know how you feel about changing the source material that much

Reply:
Also, you can tell my english is on point by my use of "seems to seem..."

Reply:
Made a minor tweak.